ci/nightly/pipeline.template.yml

```diff
@@ -827,7 +827,7 @@ steps:
     plugins:
       - ./ci/plugins/mzcompose:
           composition: cloud-canary
-    branches: "main v*.*"
```

Comment:
This is temporary to just kick off a build, will remove this commit when it's time to commit.

Reply:
https://buildkite.com/materialize/nightlies/builds/5762

Reply:
As the nightly was green I'll be so free to remove this: https://buildkite.com/materialize/nightlies/builds/5764